api/controllers/registration.py

```diff
@@ -28,11 +28,14 @@ def register_user(user_data: s.RegistrationIn, db: Session) -> s.Token:
         if db.scalar(stmt) is not None:
             raise HTTPException(status_code=status.HTTP_409_CONFLICT, detail="This email is already registered")
 
+    # validate password
+    password = s.RegistrationIn.password_validation(user_data.password)
```

Comment:
Lets do this validation at the beginning of `register_user`. There is no point in any DB queries if password fails to pass validation